docs/transfer-ledger-create.html

```diff
@@ -293,39 +304,47 @@
        return $('.form').form('is valid');
      }
      
+     function dimBody(msg){
+        $("body").dimmer({closable:false}).dimmer("show");
+        $(".dimmer").html("<div class='content'> <h3> " + msg + "</h3></div>");
+     }
+     
      async function getLedger(){
         if (!window.ledger) {
-          var transp = await window.TranspWeb.create();
+          var transp = await window.HttpTransp.open("http://localhost:5000/apdu");
```

Comment:
This appears to point to Speculos

Reply:
I do not have a ledger. So I had to do this.